repose-aggregator/docs/src/asciibinder/welcome/ver-9-upgrade-notes.adoc

```diff
@@ -3,6 +3,8 @@
 == Core Behavior Changes
 . Headers will no longer be split unnecessarily at the boundaries of *Repose*.
   If you were counting on this behavior for some reason see <<../recipes/header-splitting.adoc, Pre 9.0 Header Splitting>>.
+. The response status line reason phrase from the origin service will now be forwarded from the origin service.
+  If you were expecting *Repose* to normalize the reason phrase, that will no longer be the case.
```

Comment:
There should probably be a note here like for the Header Splitting above that points to the recipe for the Scripting filter to get the old behavior.

Reply:
Oops. Done!